src/components/bcros/businessDetails/Links.vue

```diff
@@ -63,7 +63,7 @@ const setShowDissolutionDialog = (show: boolean) => {
 }
 
 const dialogTitle = computed<string>(() => {
-  return (!currentBusiness?.value?.goodStanding && hasRoleStaff)
+  return showChangeNotInGoodStandingDialog.value
```

Comment:
I don't understand how a variable for whether to show a dialog can be used to select one other other string (or template, below).

I cannot verify this as I cannot get to the dissolution version of this dialog.